.github/workflows/build_and_deploy_hold.yml

```diff
@@ -44,16 +44,36 @@ jobs:
           echo $MAJOR_VERSION
           echo ::set-output name=major_version::${MAJOR_VERSION}
 
+  should_deploy_major_version:
+    runs-on: ubuntu-latest
+    outputs:
+      deploy_major_version: ${{ steps.vars.outputs.deploy_major_version }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: check for possible next minor version from current tag version
+        id: vars
+        run: |
+          MINOR_VERSION=$(echo "${{ github.ref_name }}" | cut -d '.' -f 2) 
+          MAJOR_VERSION=$(echo "${{ github.ref_name }}" | cut -d '.' -f 1)
+          NEXT_MINOR_VERSION=$(( $MINOR_VERSION + 1 ))
+          OUTPUT=$(git tag -l "$MAJOR_VERSION.$NEXT_MINOR_VERSION.*")
```

Comment:
Does this need 'v' in front of the major version?

Reply:
git tag would need 'v' and MAJOR_VERSION should already include 'v' since it just the substring cut off from the first dot in the tag.